AZ3166/src/libraries/AzureIoT/examples/DevKitTranslator/DevKitTranslator.ino

```diff
@@ -181,7 +181,7 @@ static void DoRecording()
 
 static void DoRecorded()
 {
-  Audio.getWav(&wavFileSize);
+  wavFileSize = Audio.getCurrentSize();

```

Comment:
please check the **WAV format data scenario** in https://microsoft.github.io/azure-iot-developer-kit/docs/apis/audioV2/

Reply:
Got it, thanks a lot.
And for this change, we have referred to the above article, we will double check it.